src/app.rb

```diff
@@ -0,0 +1,64 @@
+require_relative 'storage'
+require_relative 'actions/utiles_helper'
+
+class App
+  def initialize
+    @store = Store.new
+    @B_utiles = BookUtiles.new(@store)
+    @M_utiles = MusicUtiles.new(@store)
+    @Movies_utiles = MovieUtiles.new(@store)
+    @G_utiles = GameUtiles.new(@store)
+  end
+
+  def sub_processor(option)
+    case option
+    when 1
+      @store.books.each do |book|
+        puts "id: #{book.id}, Publisher: #{book.publisher}, Publish Date: #{book.publish_date}, Cover State: #{book.cover_state}, label: #{book.label}"
+      end
+    when 2
+      @store.musics.each do |music|
+        puts "Title: #{music.title}, Publish Date: #{music.publish_date}, on Spotify: #{music.on_spotify}, genre: #{music.genre}"
+      end
+    when 3
+      @store.movies.each do |movie|
+        puts "Silent: #{movie.silent}, Publish Date: #{movie.publish_date}, source: #{movie.source}"
+      end
+    when 4
+      @store.games.each do |game|
+        puts "Mutiplayer: #{game.mutiplayer}, Last Played At: #{game.last_played_at}, Author: #{game.author}"
+      end
```

Comment:
When I list games I get an error.

![image](https://user-images.githubusercontent.com/38088818/202694039-73767773-81da-4f38-92a3-42d3e1f6d81c.png)

- Kindly look into it. Looks like there's something wrong with the `multiplayer` method.